src/sagemaker_containers/collections.py

```diff
@@ -10,19 +10,28 @@
 # distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+from __future__ import absolute_import
 
+import collections
 
-def split_by_criteria(dictionary, keys):  # type: (dict, set) -> (dict, dict)
+SplitByCriteriaSpec = collections.namedtuple('SplitByCriteriaSpec', 'included not_included')
```

Comment:
How about "SplitResult" instead?

And "not_included" should be "excluded".

Reply:
Agreed.